docs/docs/self-host/self-host-lightdash.mdx

```diff
@@ -6,28 +6,29 @@ sidebar_position: 1
 
 Lightdash is MIT licensed and open source. You can self-host Lightdash on your own infrastructure.
 
-This guide is designed for DevOps Engineers that are familiar with Docker, Kubernetes, and are comfortable 
+This guide is designed for DevOps Engineers that are familiar with Docker, Kubernetes, and are comfortable
 configuring environment variables, SMTP credentials, and database connections.
 If you're unsure whether to self-host please read our guide on [Lightdash Cloud vs. Self-Hosted](./lightdash-cloud-vs-self-hosted.mdx).
 
 ## Prerequisites for self-hosting
 
- - Access to a kubernetes cluster and [`kubectl`](https://kubernetes.io/docs/tasks/tools/) installed
- - [Helm](https://helm.sh/docs/intro/quickstart/)
- - Docker
+- Access to a kubernetes cluster and [`kubectl`](https://kubernetes.io/docs/tasks/tools/) installed
+- [Helm](https://helm.sh/docs/intro/quickstart/)
+- Docker
 
 ## Self-host Lightdash on Kubernetes
 
 The following steps will create a Lightdash instance and a postgres database to store your metadata (note: this is
 separate to your data warehouse with your analytics data).
 
-We recommend using kubernetes + helm but you can alternatively follow these guides for a minimum deployment without 
+We recommend using kubernetes + helm but you can alternatively follow these guides for a minimum deployment without
```

Comment:
```suggestion
- We recommend using kubernetes + helm but you can alternatively follow these guides for a minimum deployment without
```

Reply:
Opted not to follow this suggestion since it looks confusing with the list just below it.